e2e/analytics-script.spec.ts

```diff
@@ -1,15 +1,11 @@
 import { test, expect } from '@playwright/test'
 
-const pages = [
-    '/',
-    '/#/jobs'
-]
 
-for (const path of pages) {
-    test(`simple analytics script present on ${path}`, async ({ page }) => {
-        await page.goto(path)
+test('SA script stays single across SPA navigations', async ({ page }) => {
+    await page.goto('/', { waitUntil: 'domcontentloaded' })
 
-        const script = page.locator('body script[src*="simpleanalyticscdn.com"]')
-        await expect(script).toHaveCount(1)
-    })
-}
\ No newline at end of file
+    for (const path of ['/#/jobs', '/#/team', '/#/events', '/#/wiki', '/#/codeofconduct']) {
```

Comment:
The `pages` variable might be good to add back, as it's easy to edit if we extend the site.